bootstrap.d/net-print.yml

```diff
@@ -0,0 +1,56 @@
+packages:
+  - name: cups
+    metadata:
+      summary: The Common Unix Printing System
+      description:  The Common Unix Printing System (CUPS) is a print spooler and associated utilities. It is based on the "Internet Printing Protocol" and provides printing services to most PostScript and raster printers.
+      spdx: 'Apache-2.0'
+      website: 'https://www.cups.org/'
+      maintainer: "Dennis Bonke <dennis@managarm.org>"
+      categories: ['net-print']
+    source:
+      subdir: ports
+      git: 'https://github.com/apple/cups.git'
+      tag: 'v2.3.3'
+      version: '2.3.3'
+      tools_required:
+        - host-autoconf-v2.69
+        - host-automake-v1.15
+      regenerate:
+        - args: ['cp',
+            '@BUILD_ROOT@/tools/host-automake-v1.15/share/automake-1.15/config.sub',
+            '@THIS_SOURCE_DIR@/']
+    tools_required:
+      - system-gcc
+      - virtual: pkgconfig-for-target
+        triple: "@OPTION:arch-triple@"
+    pkgs_required:
+      - mlibc
+      - gnutls
+      - libiconv
+      - zlib
+    configure:
+      - args: ['cp', '-r', '@THIS_SOURCE_DIR@/.', '@THIS_BUILD_DIR@']
+      - args:
+        - './configure'
+        - '--host=@OPTION:arch-triple@'
+        - '--prefix=/usr'
+        - '--libdir=/usr/lib'
+        - '--disable-systemd'
+        - '--with-rcdir=/tmp/cupsinit'
+        - '--with-system-groups=lpadmin'
+        - '--with-docdir=/usr/share/cups/doc-2.3.3op2'
+        - '--disable-dbus'
+        - '--disable-gssapi'
+        - '--disable-pam'
+        - '--disable-avahi'
+        - '--disable-dnssd'
+        - '--disable-browsing'
+        - '--with-components=libcups'
+      - args: 'sed -i s/-Werror//g @THIS_BUILD_DIR@/Makedefs'
```

Comment:
The configure script does not allow disabling `-Werror`? Likewise, patching the source does not work?

Reply:
Unfortunately not, even gentoo applies a sed to fix this issue.